### ISSUE TITLE
Option to disable single tap menu on reader 

### ISSUE THREAD

**Issue by wakoko79** [mihonapp/mihon#990]
### Describe your suggested feature

I would like an option to disable single taps when reading.
I only do scroll, so disabling tap zones is appreciated. But by doing so, a single tap anywhere will show the menu. This happens every time i scroll, even though the "sensitivity for hiding menu on scroll" is in the highest. It is very irritating, so please add this feature.

Maybe just make menu showing only trigger on back button?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Bust3t** [comment]
I would also like that option. It happens to me all the time. And the worst part is that I always hit the page select slider changing the page somewhat random.
Had the sensitivity on low and very high.
It's mostly an issue when laying down an grabbing the phone on the lower edge.

**Comment by Animeboynz** [comment]
In all honesty tho, people just need to learn how to scroll.
Touch and let go in relatively the same place -> menu appears
Touch slide to a different position and let go -> menu does not appear

https://github.com/user-attachments/assets/9f80681e-9ec5-4387-b9e6-8a652ad9b326

**Comment by Animeboynz** [comment]
> They will swipe up and tap the middle of the screen to stop, I don't know where they learned this behaviour

That is one of the best ways to stop scrolling immediately when you are trying to scroll really fast.
but even that does not trigger the menu though 

https://github.com/user-attachments/assets/f642e83b-0147-4239-a172-60f30cda3dc7

**Comment by wakoko79** [comment]
> What should really be asked for is not to disable the zone cause that would make you stuck on the reading page, but a tap layout that doesn't have the "menu" button area in the middle but at the top or smth. with nothing in the middle

Yeah, putting the tap zone for single tap menu reveal on top area should be fine.

> So you swipe up, stop, then raise your thumb, no need to tap the middle. I know it's like magic, genius even.

Have you tried doing that on a spread page while zoomed in? Going from top to bottom and back so you can read speech bubbles? It will take you like 5 scrolls with that.   

That is why people do swipe - let go - tap to brake.   
You do an action appropriate for what you want to do instead of trying to fit an action to what you want to do.

**Comment by scb261** [comment]
> Yeah, putting the tap zone for single tap menu reveal on top area should be fine.

Have you explored available tap zones? There is one that does exactly that

**Comment by wakoko79** [comment]
> [@wakoko79](https://github.com/wakoko79) yes you are the only one who has ever zoomed in and swiped up. You are very special.

Why so angry? I was just explaining a use-case for tap braking.
This thread is just a suggestion. Ignore it if you want. No need to act like child.

> Have you explored available tap zones? There is one that does exactly that

My tap zones are disabled, I don't wanna tap. I'm also on long strip mode. 
  

If we can have a dedicated tap zone for menu reveal, instead of it being the default empty area (which are not occupied by prev or next page tap zones), then it would be great.  


This also happens to me when menu shows up:  
> I would also like that option. It happens to me all the time. And the worst part is that I always hit the page select slider changing the page somewhat random. Had the sensitivity on low and very high. It's mostly an issue when laying down an grabbing the phone on the lower edge.
